### :x: 2 Tests Failed:
| Tests completed | Failed | Passed | Skipped |
|---|---|---|---|
| 2758 | 2 | 2756 | 30 |
<details><summary>View the top 2 failed tests by shortest run time</summary>

> 
> ```
> github.com/status-im/status-go/peers TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery
> ```
> 
> <details><summary>Stack Traces | 20s run time</summary>
> 
> > 
> > ```
> > === RUN   TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery&#xA;    peerpool_test.go:101: &#xA;        &#x9;Error Trace:&#x9;.../status-go_prs_tests_PR-5844/peers/peerpool_test.go:101&#xA;        &#x9;            &#x9;&#x9;&#x9;&#x9;.../status-go_prs_tests_PR-5844/peers/peerpool_test.go:378&#xA;        &#x9;Error:      &#x9;timed out waiting for a peer&#xA;        &#x9;Test:       &#x9;TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery&#xA;    peerpool_test.go:379: &#xA;        &#x9;Error Trace:&#x9;.../status-go_prs_tests_PR-5844/peers/peerpool_test.go:379&#xA;        &#x9;Error:      &#x9;Not equal: &#xA;        &#x9;            &#x9;expected: &#34;6af7eca59a1b1560592f3ddf7c961103ce6753e0ec9ba242054a0048b30b8437&#34;&#xA;        &#x9;            &#x9;actual  : &#34;0000000000000000000000000000000000000000000000000000000000000000&#34;&#xA;        &#x9;            &#x9;&#xA;        &#x9;            &#x9;Diff:&#xA;        &#x9;            &#x9;--- Expected&#xA;        &#x9;            &#x9;+++ Actual&#xA;        &#x9;            &#x9;@@ -1 +1 @@&#xA;        &#x9;            &#x9;-6af7eca59a1b1560592f3ddf7c961103ce6753e0ec9ba242054a0048b30b8437&#xA;        &#x9;            &#x9;+0000000000000000000000000000000000000000000000000000000000000000&#xA;        &#x9;Test:       &#x9;TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery&#xA;    peerpool_test.go:112: &#xA;        &#x9;Error Trace:&#x9;.../status-go_prs_tests_PR-5844/peers/peerpool_test.go:112&#xA;        &#x9;            &#x9;&#x9;&#x9;&#x9;.../status-go_prs_tests_PR-5844/peers/peerpool_test.go:382&#xA;        &#x9;Error:      &#x9;timed out waiting a pool event&#xA;        &#x9;Test:       &#x9;TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery&#xA;--- FAIL: TestPeerPoolSimulationSuite/TestMailServerPeersDiscovery (20.02s)&#xA;
> > ```
> 
> </details>


> 
> ```
> github.com/status-im/status-go/peers TestPeerPoolSimulationSuite
> ```
> 
> <details><summary>Stack Traces | 20s run time</summary>
> 
> > 
> > ```
> > === RUN   TestPeerPoolSimulationSuite&#xA;--- FAIL: TestPeerPoolSimulationSuite (20.04s)&#xA;
> > ```
> 
> </details>

</details>

To view individual test run time comparison to the main branch, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/status-im/status-go/tests/fix-sqlcipher-for-ndk-r26b)

--- a/go.mod
+++ b/go.mod
@@ -14,7 +14,7 @@ replace github.com/nfnt/resize => github.com/status-im/resize v0.0.0-20201215164
 
 replace github.com/forPelevin/gomoji => github.com/status-im/gomoji v1.1.3-0.20220213022530-e5ac4a8732d4
 
-replace github.com/mutecomm/go-sqlcipher/v4 v4.4.2 => github.com/status-im/go-sqlcipher/v4 v4.5.4-status.2
+replace github.com/mutecomm/go-sqlcipher/v4 v4.4.2 => github.com/status-im/go-sqlcipher/v4 v4.5.4-status.3
 
 replace github.com/libp2p/go-libp2p-pubsub v0.11.0 => github.com/waku-org/go-libp2p-pubsub v0.0.0-20240703191659-2cbb09eac9b5
 

